spring-cloud-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/NacosDiscoveryProperties.java

```diff
@@ -519,7 +531,48 @@ private Properties getNacosProperties() {
 		properties.put(SECRET_KEY, secretKey);
 		properties.put(CLUSTER_NAME, clusterName);
 		properties.put(NAMING_LOAD_CACHE_AT_START, namingLoadCacheAtStart);
+
+		enrichNacosProperties(properties);
 		return properties;
 	}
 
+	private void enrichNacosProperties(Properties properties) {
+		Map<String, String> configurationItem = getConfigurationItemFromEnv();
+		configurationItem.forEach((k, v) -> {
+			if (!properties.contains(k)) {
+				properties.put(k, v);
+			}
+		});
```

Comment:
May be can simplify the code, `properties::putIfAbsent`

Reply:
has been resolved it. please review it again